tests/build-job-matrix.py

```diff
@@ -0,0 +1,120 @@
+import argparse
```

Comment:
would like to see this added to the internal actions repo as its own job maybe. not sure. since it is a specific python job for this task. Leaving this comment to spur discussion.

Reply:
I think it might be kind of tricky to make this an action as a python script.. Actions I think typically only support javascript (or else, more basic yaml). Might be nice to make this generic enough to be an action and we can reuse it elsewhere down the road, but I think I'd be ok if this goes in chia-blockchain for now and we can abstract/move it when we need it somewhere else? I think it's basically just replacing a similar script that used to live here anyways.